resolwe_bio/tools/rnaseq.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+"""RNA-Seq for biologists."""
```

Comment:
Either use title case "RNA-Seq for Biologists" or sentence case "RNA-seq for biologists".